scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: ardana-ccp-c0-m1-mgmt
+  gather_facts: False
+  include_vars: "provider.yml"
+  vars:
+    caasp_image_url: "http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    caasp_image_path: "/tmp/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+  tasks:
+  - name: Install caasp-openstack-heat-templates
+    always_run: True
+    become: yes
+    zypper:
+      name: caasp-openstack-heat-templates
+      state: present
+
+  - name: Download CaaSP image
+    always_run: True
+    get_url:
+      url: "{{caasp_image_url}}"
+      dest: "{{caasp_image_path}}"
+
+  - name: Upload CaaSP image to glance
+    always_run: True
+    command: openstack {{ openstack_cmd_args_g }} image create --public --disk-format qcow2 --container-format bare --file "{{caasp_image_path}}" CaaSP-3
```

Comment:
nit: Both openstack_cmd_args_g  and openstack_cmd_args_h could be removed if clouds.yaml was implemented, and pointing to the appropriate cloud. The less code to carry, the better.

nit2: Could wrap caasp_image_path with spaces for consistency: "{{ caasp_image_path }}"

Reply:
Will check the cloud.yaml bit but I think if we go with creating a separate playbook for caasp we will need this..atleast that's how the other playbooks did it.